cea/optimization/master/summarize_network.py

```diff
@@ -11,9 +11,9 @@
 import pandas as pd
 import math
 
-__author__ = "Jimeno A. Fonseca"
-__copyright__ = "Copyright 2015, Architecture and Building Systems - ETH Zurich"
-__credits__ = ["Jimeno A. Fonseca", "Thuy-An Nguyen", "Tim Vollrath", ]
+__author__ = "Sreepathi Bhargava Krishna"
```

Comment:
Modified. I was actually thinking this was to contact the person who has lastly done the changes to it. But it was foolish of me. Reverted it back